usaspending_api/api_contracts/contracts/v2/search/spending_by_category.md

```diff
@@ -115,16 +114,7 @@ This endpoint returns a list of the top results of specific categories sorted by
   If the `def_codes` provided are in the COVID-19 or IIJA group, the query will only return results of transactions where the `action_date` is on or after the enactment date of the public law associated with that disaster code.
 
 ### TimePeriodObject (object)
-+ `start_date` (required, string)
-    Currently limited to an earliest date of `2007-10-01` (FY2008).  For data going back to `2000-10-01` (FY2001), use either the Custom Award Download
-    feature on the website or one of our `download` or `bulk_download` API endpoints.
-+ `end_date` (required, string)
-    Currently limited to an earliest date of `2007-10-01` (FY2008).  For data going back to `2000-10-01` (FY2001), use either the Custom Award Download
-    feature on the website or one of our `download` or `bulk_download` API endpoints.
-+ `date_type` (optional, enum[string])
-    + Members
-        + `action_date`
-        + `last_modified_date`
+See [TimePeriodObject](./spending_by_category/)
```

Comment:
Is this the correct link?

Reply:
Yes this is the correct link. Each category has its own time period object. The intent of line 117 is to say go look at the category of interest TimePeriodObject. It will tell you more about the TimePeriodObject

Reply:
Gotcha, makes sense. Can you add a little more explanation here? Maybe:
```
See [TimePeriodObject](./spending_by_category/) definition for individual categories
```